Update requirements.txt to accept more python/vtk versions

pieper opened the pull request:
Possible fix for the issue reported here:

https://discourse.slicer.org/t/slicer-doesnt-open-after-installing-slicerwma-extension/42356

pieper committed:
Update requirements.txt to accept more python/vtk versions

Possible fix for the issue reported here:

https://discourse.slicer.org/t/slicer-doesnt-open-after-installing-slicerwma-extension/42356

jhlegarreta reviewed (commented):
Have not tested locally. I am fine with these as long as the CIs are passing. Thanks Steve.

pieper committed:
Update test_package.yaml - remove python 3.8

Python 3.8 is end-of-life and no longer supported.

https://devguide.python.org/versions/

jhlegarreta reviewed (commented):
Dropping a Python version means a major version update:
https://github.com/SlicerDMRI/whitematteranalysis/blob/master/setup.py#L35

So, it would mean a 1.0.0 version.

Also, I see that there is no `python_requires` section in the `setup.py` file. Maybe it is a good time to do that.

Can these changes be made?

pieper commented:
@jhlegarreta the authors of this extension have no time to help with this right now and @ljod would like to have a working version for the Slicer extension to correct a serious regression.  If you have time to implement your suggestions please do, otherwise let's merge this.

ljod reviewed (approved):
Looks good thanks Steve
